service/switchboardapp/src/main/java/com/switchboard/switchboardapp/SwitchboardappApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.switchboard.switchboardapp;
```

Comment:
Package name is repetitive, maybe `com.switchboard.app`?

Reply:
Fixed.